docker-compose.yml

```diff
@@ -49,9 +69,14 @@ services:
     volumes:
       # Mount the app/ directory so live reload works
       - ./app:/app/app:ro
+    healthcheck:
+      test: wget http://frontend:4200/ -O /dev/null
+      interval: 5s
+      start_period: 120s
```

Comment:
I'm now wondering if we actually need these healthchecks for the `frontend` and `backend` containers. I guess the `worker` container only really needs to the `postgres` container too. 🤔

Reply:
the reason why I'm asking is that the `curl http://backend:8888/api/v1/site_metadata` above makes the console quite noisy due to the repeated HTTP requests that are coming in from the health check.

Reply:
> makes the console quite noisy

Yeah, I did notice that. I'm actually not sure if any of the healthchecks are needed anymore, from what I understand depends-on does not wait for the service to be healthy, only for the container to be started. And also that docker uses healthcheks only to report the status of the service, but I believe it takes no action when a service stops being "healthy" (unless docker swarm is used, I think?)

https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on
> There are several things to be aware of when using depends_on:
> depends_on does not wait for db and redis to be “ready” before starting web - only until they have been started. If you need to wait for a service to be ready, see Controlling startup order for more on this problem and strategies for solving it.

https://docs.docker.com/engine/reference/builder/#healthcheck
> When the health status of a container changes, a health_status event is generated with the new status.

Not exactly an authority in this case, posting it for more context
https://stackoverflow.com/questions/47088261/restarting-an-unhealthy-docker-container-based-on-healthcheck

Reply:
yeah, that makes sense. if it works fine  without the healthchecks then I'd say let's remove them. I initially added them because I clearly didn't know what I was doing :D

Reply:
I get it, I also don't understand why did they (docker people) leave healthchecks in this half-useless state (or maybe I'm completely misunderstanding something), I remember having a similar issue in a different project I worked on.

It seems to work fine without healthchecks, I pushed the latest commit.